cfgov/jinja2/v1/activity-log/_activity-list.html

```diff
@@ -23,12 +23,12 @@
                     {% if is_blog(post) %}
                         {{ category_slug.render('blog',
                                 page_url,
-                                'meta-header_left',
+                                'm-meta-header_left',
```

Comment:
Hmm... we perhaps shouldn't have added `float: left;` to `meta-header_left` [when making it atomic](https://github.com/cfpb/capital-framework/commit/6662506bbfcea95f739e9e0c1689ec4fac83b908#diff-fbe01bf95b92595df0d955810b71204dR11)

Having that be in there results in this unsightliness:

![screen shot 2017-07-11 at 11 40 33](https://user-images.githubusercontent.com/1044670/28077141-dcb55504-662e-11e7-8ad0-ed4a6b73856d.png)

It does nothing of value right now, so as a temporary measure, I'd recommend removing the `m-meta-header_left` classes from the resulting markup.